SimPEG/regularization/base.py

```diff
@@ -16,19 +16,32 @@
 
 
 class BaseRegularization(BaseObjectiveFunction):
-    """
-    Base class for regularization. Inherit this for building your own
-    regularization. The base regularization assumes a weighted l2-norm style of
-    regularization. However, if you wish to employ a different norm, the
-    methods :meth:`__call__`, :meth:`deriv` and :meth:`deriv2` can be
-    over-written
-
-    :param discretize.base.BaseMesh mesh: SimPEG mesh
-    :param active_cells: Array of bool defining the set of active cells.
-    :param mapping: Model map
-    :param reference_model: Array of model values used to constrain the inversion
-    :param units: Model units identifier. Special case for 'radian'
-    :param weights: Weight multipliers to customize the least-squares function.
+    """Base regularization class.
+
+    The ``BaseRegularization`` class defines properties and methods inherited by
+    SimPEG regularization classes. It is not directly used to constrain inversions.
```

Comment:
```suggestion
    SimPEG regularization classes. It is not directly used in an inversion, but rather inherited.
```

small suggestion (the code is technically `used` but we don't recommend that users work with it directly)

Reply:
Done